adb_graphics/specs.py

```diff
@@ -402,9 +402,12 @@ def vis_colors(self) -> np.ndarray:
         ifr = cm.get_cmap('autumn', 30)(range(0, 30))
         mvfr = cm.get_cmap('Blues', 20)(range(10, 20))
         vfr1 = cm.get_cmap('YlGn_r', 60)(range(0, 50))
-        vfr2 = cm.get_cmap('Reds', 15)(np.full(51, 1))
+        vfr2 = cm.get_cmap('Greys', 25)(np.full(10, 9))
+        hi01 = cm.get_cmap('Greys', 25)(np.full(10, 6))
+        hi02 = cm.get_cmap('Greys', 25)(np.full(20, 3))
+        hi03 = cm.get_cmap('Greys', 25)(np.full(1, 0))
 
-        return np.concatenate((lifr, ifr, mvfr, vfr1, vfr2))
+        return np.concatenate((lifr, ifr, mvfr, vfr1, vfr2, hi01, hi02, hi03))
```

Comment:
Did Stan provide any categories for these additional gray ranges? It might be useful to add those to the doc block above. If he didn't, perhaps adding a line to indicate that an arbitrary gray scale covers values with greater than 5 mi (?) of visibility. "Arbitrary" would be important here since the other categories have solid rules in the aviation world.

Reply:
added this to the vis_colors method in specs.py:

"the gray range is arbitrary compared to the official flight levels"